avocado/utils/linux_modules.py

```diff
@@ -39,17 +39,18 @@
 BUILTIN = 2
 
 
-def load_module(module_name):
+def load_module(module_name, parameters=''):
     """
     Checks if a module has already been loaded.
     :param module_name: Name of module to check
+    :param parameters: Additional parameters to load module with
     :return: True if module is loaded, False otherwise
     :rtype: Bool
     """
     if module_is_loaded(module_name):
         return False
 
-    process.system('/sbin/modprobe ' + module_name)
+    process.system('/sbin/modprobe %s %s' % (module_name, parameters))
```

Comment:
@ldoktor Thanks for the review. Please find my comments below:

1. I got this :-). Have a working copy of it, just that I forgot to include this into this PR.
2. It is difficult, since the actual change happens in /sys/class/<pci class>/<interface>/parameter, and we need to split the parameter key and value to check this. I consider it is a bit too much to take care of in the utility, and would be better to take care of in a test, which I am coming up with.
3. Is that the preferred method, and not via modprobe ?
4. I am working on the exact same scenario for a test, and would consider this.
5. It is a bit tricky, since loading a module with a parameter value almost always returns success (even for unsupported values). So, it is very difficult to implement, IMO.